code/datums/skills/combat.dm

```diff
@@ -50,24 +53,48 @@
 
 /datum/skill/combat/polearms
 	name = "Polearms"
+	dreams = list(
+		"...the air blows against your body in the open field, as your opponent's whole body is at range with the polearm's great length, regaining your focus you'd start to pull the polearm with a firm grip to ready up for a proper instead of a half-hearted hasty half-forced strike and thrusting it forth, keeping your opponent at a distance, striking where they move or at when close and with proper precision when they're at range, with double the said range came the advantage when chasing and tactically falling back from them, piercing through padded-cloth and leather armor like they were nothing with the spear, throwing it towards someone unarmored before rolling towards an heavier polearm and reaching for the heavens and with the added polearm range unleashed a slow but crashing blow through that plated armor making them scream for pain and slowly crawl towards and away, never being able to reach you back until they were properly subdued..."
+	)
 
 /datum/skill/combat/maces
 	name = "Maces"
+	dreams = list(
+		"...maces maces maces, they're all around, striking to slow, unleashing a hefty delayed smash to fully penetrate all that would rather stand in your path, stunning them briefly, smashing them when they're down to be unable to get up, what an amusing sight."
```

Comment:
Maces aren't about penetrating, they're about blunt trauma, no? As I understand they're an anti-armor weapon meant to deliver a lot of force, denting armor is a viable strategy- damaged armor can restrict the wearers range of motion, or even injure them.

Reply:
They quite literally have armor penetration 80 on smash, same as the pick intent on daggers, so, yes, they pierce armor hard, the majority of armors in the game, have not enough points to absorb the full brunt of a mace smash, you may as well be wearing nothing, for plate can't stop it from dealing full damage, the only thing it stops is getting critted by it.